### DISCUSSION TITLE
Multiple Logos in the footer - all sized similarly

### DISCUSSION THREAD

**Discussion by KJBock** [CollectionBuilder/collectionbuilder.github.io#102] [Ideas]
Hi there, 
I am using Collection Builder to display a web version of a poster exhibition. I am PhD student and the project is funded by different people, I wanted to include multiple logos/links to all the instituitions in the footer.
I also wanted to make them all the same size, without calculating pixel sizes or distorting the logos themselves. 

I defined the logos for the multiple insitutions in the config.yml and then entered them by copying the exisiting IF close for the logo. To make it the same size I defined a "style" function with a fixed width and had height on auto. (style="width:180px; height:auto")

I looked at some example pages and the logos had different sizes - so I thought this might be helpful for others, too.

**Comment by owikle** [comment]
@KJBock What a cool project and nice work figuring out this customization! Thanks for sharing! Do you mind if we post a link here to your project site and footer code so that others can easily find them?

**Reply by KJBock** [reply]
@owikle, all thanks to you and the team really :)  
Here is the link to my project site: https://kjbock.github.io/creativeresponses2pma/
How shall I share the code? (I am new to coding and I am not sure what is the easiest was, just copy+paste?)

**Reply by owikle** [reply]
Good question! You can either post links to the _config.yml and _includes/footer.html files in your repository (for example: https://github.com/KJBock/creativeresponses2pma/blob/main/_includes/footer.html), or you can copy and paste the specific parts of your code that you changed and encase it in triple backticks (```) so it shows up as raw code in your comment. Here's GitHub's documentation on how to use the backticks: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks. Thanks for being willing to share!
